### DISCUSSION TITLE
Consequences of plugging in servos/motors backwards

### DISCUSSION THREAD

**Discussion by jds-designs** [gizmo-platform/gizmo#28] [Q&A] [answered]
The document "[The Team Experience](https://dash.bestrobotics.org/download.php?id=4711&folder=1454)" has a diagram of the correct orientation to plug in cables to the Gizmo board. I just wanted to ask, how serious are the consequences of running the Gizmo with a servo or motor plugged in backwards? I want to know how much I should emphasize this when handing out returnable kits to our teams at kickoff.

**Comment by the-maldridge** [comment, marked as answer]
You should emphasize it a lot.  Plugging them in backwards shouldn't damage the Gizmo, but it may ruin a motor controller.  What can smoke the student processor is if the motor controller is plugged in not aligned correctly on the 4 pins (leftmost pin is unused for a 3 pin motor controller).

**Reply by jds-designs** [reply]
Ok, thank you for the information.

**Comment by dwfunk4475** [comment]
I can now definitively answer this question! It's catastrophic ! You will lose the Pico2!

I had passed on to the team that the solution to reversing the motors in code was really to reverse the wires to the motor.

For some reason they thought reversing the 3pin controller connector was better than unscrewing the wires on the terminal block.

Reversing the 3pin puts the ground directly on a GPIO pin. The return path for up to 8.4 volts @ 4 amps is now through the Pico2, a 3.3 volt device. (poly will most likely trip, but that's now immaterial)

We're making colored labels for marking our ports.

**Reply by the-maldridge** [reply]
You may also wish to print the updated case lid which includes keying to make sure the connector is accepted only one way.

**Reply by dwfunk4475** [reply]
Fantastic! Thanks! Did not know that was available. We will do that ASAP.
